Source/Core/VideoCommon/VertexLoaderManager.h

```diff
@@ -53,8 +53,12 @@ void UpdateVertexArrayPointers();
 
 // Position cache for zfreeze (3 vertices, 4 floats each to allow SIMD overwrite).
 // These arrays are in reverse order.
-extern float position_cache[3][4];
-extern u32 position_matrix_index[4];
+extern std::array<std::array<float, 4>, 3> position_cache;
```

Comment:
I have a general question, why do we do this?  Is there a reason we don't have all the position information available for a specific draw?  I'm assuming it is for performance / memory reasons.

I ask because I too was looking at this code but in my case I wanted to get all the vertex positions in a particular instance and modify them.  I hoped it'd be easy but there's this loader design that makes that difficult.